client.go

```diff
@@ -104,6 +110,7 @@ func (c *Client) Call(ctx context.Context, service, method string, req, resp int
 func (c *Client) dispatch(ctx context.Context, req *Request, resp *Response) error {
 	errs := make(chan error, 1)
 	call := &callRequest{
+		ctx:  context.TODO(),
```

Comment:
Should this split the context?

Reply:
I wasn't sure what to do with this, because we were passing a nil context before AFAICT.
It seems like passing through the existing context is desirable, but again I didn't know why it wasn't already.

Reply:
it seems that the timeout control is at here .https://github.com/containerd/ttrpc/blob/ce5247614dcad57f9c9685f3845428a1f22e19a0/client.go#L119-L124.

And the `send` uses the call.ctx. I think we should not split the context here, though `send` doesn't use context right now.

https://github.com/containerd/ttrpc/blob/ce5247614dcad57f9c9685f3845428a1f22e19a0/client.go#L199-L206